pom.xml

```diff
@@ -59,7 +59,7 @@ limitations under the License.
 
         <!-- google-cloud-java related dependency versions -->
         <!-- TODO: check if commons-codec was upgraded to 1.13 in org.apache.httpcomponents from http-client-->
```

Comment:
commons-codec is still 1.11 in org.apache.httpcomponents via google-http-client.